deployment/v3/mosip/pms/install.sh

```
@@ -24,39 +23,50 @@ function installing_pms() {
 
   INTERNAL_API_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-api-internal-host})
   PMP_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-pmp-host})
-  PMP_NEW_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-pmp-reactjs-ui-new-host})
+  PMP_REVAMP_UI_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-pmp-revamp-ui-host})
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)